### PR TITLE
Add possibility to bind to a specific IP address

### PR DESCRIPTION
**Pull request by TwizzyDizzy**

### CODE REVIEW

File: server/config.js [hunk 1]
```diff
@@ -14,6 +14,11 @@ const conf = convict({
     default: 'localhost',
     env: 'REDIS_HOST'
   },
+  listen_address: {
+  format: 'ipaddress',
+  default: '127.0.0.1',
```
Comment (dannycoates) [inline server/config.js]: Let's make the default `'0.0.0.0'` so that it will listen on all addresses

**Comment by TwizzyDizzy** [comment]
Changed as requested.

Cheers
Thomas

**Comment by dannycoates** [comment]
Thank you @TwizzyDizzy 😄

**Comment by TwizzyDizzy** [comment]
Closing, since merged to master.

Cheers
Thomas

**Pull request merged**